pkg/monitoring/domainstats/downwardmetrics/scraper.go

```diff
@@ -20,9 +20,11 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/stats"
 )
 
-const DownwardmetricsRefreshDuration = 5 * time.Second
-const DownwardmetricsCollectionTimeout = collector.CollectionTimeout
-const qemuVersionUnknown = "qemu-unknown"
+const (
+	DownwardmetricsRefreshDuration   = 5 * time.Second
+	DownwardmetricsCollectionTimeout = collector.CollectionTimeout
+	qemuVersionUnknown               = "qemu-unknown"
```

Comment:
How often is this used? Can it be moved closer to the place(s) where it is used?

Reply:
inlined